src/cljs/rems/user.cljs

```diff
@@ -0,0 +1,70 @@
+(ns rems.user
+  (:require [clojure.string :as str]
+            [re-frame.core :as rf]
+            [rems.atoms :refer [info-field readonly-checkbox]]
+            [rems.common.application-util :refer [get-member-name]]
+            [rems.common.util :refer [index-by]]
+            [rems.text :refer [text localized]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+(defn username [attributes]
+  (when-let [name (get-member-name attributes)]
+    [info-field (text :t.applicant-info/name) name {:inline? true}]))
+
+(defn attributes [attributes]
```

Comment:
This could have a docstring. I'm kind of missing the detail about where this is supposed to be used.